runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -69,6 +77,17 @@ public void checkForClues()
 
 		if (clueScroll == null)
 		{
+			Query inventoryQuery = new InventoryWidgetItemQuery();
+			WidgetItem[] inventoryWidgetItems = queryRunner.runQuery(inventoryQuery);
```

Comment:
You should not run this unless the current clue != null, otherwise it is pointless

Reply:
clueScroll is the text interface of the clue scroll, and players can have a clue scroll in their inventory without that interface being open.

Reply:
I mean ```clueScrollOverlay.clue```

Reply:
Ah, I'll fix that